test/upgrade/installation/serverless.go

```diff
@@ -129,6 +129,22 @@ func DowngradeServerless(ctx *test.Context) error {
 		}
 	}
 
+	dcrd := "domainmappings.serving.knative.dev"
+
+	if err := moveCRDsToAlpha(ctx, dcrd); err != nil {
+		return err
+	}
+
+	// We might have to do an empty patch as in the example bellow for all existing resources
+	//if _, err :=	ctx.Clients.Serving.ServingV1alpha1().DomainMappings("...").Patch(context.Background(), "dm_name", types.MergePatchType, []byte("{}"), metav1.PatchOptions{}, ""); err != nil {
```

Comment:
Operators will probably need to run this for their resources or delete/recreate them.

Reply:
The reason for the error in tests is that in 1.10 (1.31) we have v1beta1 being now the stored version.

```    
    - name: v1alpha1
      served: true
      storage: true
    - name: v1beta1
      served: true
      storage: false 
```

while in 1.11 (1.32)

```    
    - name: v1beta1
      served: true
      storage: true
    - name: v1alpha1
      served: true
      storage: false
```